{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Storage 3",
      "fix_state": "Under investigation",
      "package_name": "cloud-init",
      "cpe": "cpe:/a:redhat:storage:3"
    },
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Under investigation",
      "package_name": "cloud-init",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Under investigation",
      "package_name": "cloud-init",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Under investigation",
      "package_name": "cloud-init",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Virtualization 4",
      "fix_state": "Under investigation",
      "package_name": "cloud-init",
      "cpe": "cpe:/:redhat:enterprise_linux:::hypervisor"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2020-02-05T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-8631 cloud-init: Use of unsafe SystemRandom when generating random password",
    "id": "1798731",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1798731"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "2.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:H/PR:N/UI:R/S:U/C:L/I:N/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-330",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2020-8631",
  "document_distribution": "",
  "details": [
    "cloud-init through 19.4 relies on Mersenne Twister for a random password, which makes it easier for attackers to predict passwords, because rand_str in cloudinit/util.py calls the random.choice function."
  ],
  "references": null
}